onnxruntime/test/testdata/matmul_integer_to_float.py

```diff
@@ -95,6 +106,25 @@ def GenerateModel(model_name, sign_i, sign_w, output_type_fp16, has_zp=True, bia
         has_zp=False,
         bias=True,
     )
+    GenerateModel(
+        "matmul_integer_to_float_int8_bias_initializer.onnx",
+        sign_i=False,
+        sign_w=True,
+        output_type_fp16=False,
+        has_zp=False,
+        bias=True,
+        bias_initializer=True,
+    )
+    GenerateModel(
+        "matmul_integer_to_float_int8_bias_flipped_initializer.onnx",
```

Comment:
Nit: Atleast to me, it is not very intuitive to understand that "bias flipped" means the ordering of the bias input into the "Add" node is different. Maybe using something explicit like `matmul_integer_to_float_int8_bias_**index0**_initializer` might communicate the point better.